userdata/userdata.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Steffen Vogel.
```

Comment:
Here and elsewhere: this needs to be Copyright Google LLC, not any individual’s name

Reply:
Mhh, okay. I disagree with this as in Germany, only individuals can be the owner of a copyrighted work.
And I already have signed a CLA with which I provide Google all rights to use my contribution.

Nevertheless, I changed it..